File: backend/src/main/scala/kpn/api/common/status/LogPage.scala
package kpn.api.common.status

case class LogPage(

  timestamp: ActionTimestamp,

  periodType: String,
  periodTitle: String,
  previous: String,
  next: String,

  tile: BarChart,
  tileRobot: BarChart,
  api: BarChart,
  apiRobot: BarChart,
  analysis: BarChart,
  analysisRobot: BarChart,
  robot: BarChart,
  nonRobot: BarChart
)

File: backend/api/src/main/java/kpn/api/common/status/LogPage.java
package kpn.api.common.status;

import kpn.api.common.status.ActionTimestamp;
import kpn.api.common.status.BarChart;

public record LogPage(
  ActionTimestamp timestamp,
  String periodType,
  String periodTitle,
  String previous,
  String next,
  BarChart tile,
  BarChart tileRobot,
  BarChart api,
  BarChart apiRobot,
  BarChart analysis,
  BarChart analysisRobot,
  BarChart robot,
  BarChart nonRobot
) {}

